dev/MRTCore/build-mrt.yml

```diff
@@ -94,6 +94,13 @@ steps:
     arguments: '${{ parameters.MRTSourcesDirectory }}'
     modifyEnvironment: true
 
+- task: powershell@2
+  displayName: 'Download .net Windows runtime installer'
```

Comment:
I assume this isn't actually about winrt, consider reversing terms here to avoid confusion (download .NET runtime Windows installer)